scripts/zkai_d128_gate_value_activation_fused_gate.py

```
@@ -0,0 +1,556 @@
+    if accounting.get("component_sum_bytes") != expected["local_typed_bytes"]:
+        raise FusedGateError(f"{expected['role']} component sum drift")
+    grouped = require_dict(accounting.get("grouped_reconstruction"), "grouped reconstruction")
+    if grouped != EXPECTED_GROUPED[expected["role"]]:
+        raise FusedGateError(f"{expected['role']} grouped reconstruction drift")
+
+
+def validate_envelope(path: pathlib.Path, expected: dict[str, Any]) -> dict[str, Any]:
+    try:
+        envelope_size = path.stat().st_size
+    except OSError as err:
+        raise FusedGateError(f"failed to stat {expected['role']} envelope: {err}") from err
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)